### PR TITLE
121945923 Shop owners add products

### PR DESCRIPTION
**Pull request by nwochaadim**
## What does this PR do?

Ensures that shop owners are able to add products
## Description of task to be completed?

Shop owners should be able to add products with information about the product
## What are the relevant pivotal tracker stories

121945923

### CODE REVIEW

File: spec/controllers/products_controller_spec.rb [hunk 1]
```diff
@@ -20,4 +22,32 @@
...
+    end
+  end
+
+  describe "#create" do
+    context "when all required product details are filled" do
+      it "redirects to the shop_products path" do
+        product = build(:product)
+        params = { product: product.attributes }
+        post :create, params
+        expect(response).to redirect_to(shop_products_path(@shop_owner))
```
Comment (nwochaadim) [inline spec/controllers/products_controller_spec.rb]: Make it one line with the params

File: spec/controllers/products_controller_spec.rb [hunk 2]
```diff
@@ -20,4 +22,32 @@
...
+      end
+    end
+
+    context "when all required product details are not filled" do
+      it "renders the new template with form errors" do
+        product = build(:product, quantity: nil)
+        invalid_product_attributes = product.attributes
+        params = { product: invalid_product_attributes }
+        post :create, params
```
Comment (nwochaadim) [inline spec/controllers/products_controller_spec.rb]: Same with above

**Comment by betkom** [comment]
This looks good to merge @andela-anwocha 👍

**Comment by betkom** [comment]
Can't merge, cos I don't write access to this repo cc @andela-bmakinwa

**Comment by makinwab** [comment]
@andela-tkomolafe you should be able to merge now.

**Pull request merged**